lib/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegates_bloc.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:bloc/bloc.dart';
+import 'package:equatable/equatable.dart';
+import 'package:seeds/datasource/remote/model/delegate_model.dart';
+import 'package:seeds/datasource/remote/model/member_model.dart';
+import 'package:seeds/domain-shared/page_command.dart';
+import 'package:seeds/domain-shared/page_state.dart';
+import 'package:seeds/domain-shared/result_to_state_mapper.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/mapper/delegates_load_data_mapper.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/mapper/remove_delegate_mapper.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/usecase/delegates_load_data_usecase.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/usecase/load_delegate_as_member_use_case.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/usecase/remove_delegate_use_case.dart';
+import 'package:seeds/screens/explore_screens/vote_screens/delegate/delegates_tab/interactor/viewmodels/delegates_page_commands.dart';
+
+part 'delegates_event.dart';
+part 'delegates_state.dart';
+
+class DelegatesBloc extends Bloc<DelegatesEvent, DelegatesState> {
+  DelegatesBloc() : super(DelegatesState.initial()) {
+    on<LoadDelegatesData>(_loadDelegatesData);
```

Comment:
first time i see these. What are they for? whats going on here

Reply:
oh, those are events. Are you are firing private functions on each event.

Reply:
This is different from what we have been doing so far. 

If we are to change a patter in the way we handle events in bloc, lets bring it up to the team. And make sure everyone is aware. 

Consistency is important.

Reply:
I mention this in the PR's description.
It's the new Bloc Api noting braking just changes in bloc clases.